src/openrct2/actions/FootpathPlaceFromTrackAction.cpp

```diff
@@ -30,6 +30,14 @@ FootpathPlaceFromTrackAction::FootpathPlaceFromTrackAction(
 {
 }
 
+void FootpathPlaceFromTrackAction::AcceptParameters(GameActionParameterVisitor& visitor)
```

Comment:
Remove this. This action should ONLY be used by track designs and never by plugins.